Before: be4e4e02e6bece079c79d9ab595e8e9452968b1fc5f2bc444423e6ff105b43ef
After: a75b7e5c2cf3b7a1458fc55ea89f0883c821ed0b4fcb8d655d0a13d98f9cb0ae
Patch: 14f1f43ac17ef1e6f56dbf52c2c110582ddfbbd02a4205ebac453bb186ba4f3d
--- lib/corvid/%name%_plugin.rb.tt	2012-08-14 11:23:29.836643613 +1000
+++ lib/corvid/%name%_plugin.rb.tt	2012-08-14 11:23:56.566790574 +1000
@@ -1,24 +1,9 @@
 require 'corvid/plugin'
-#require 'corvid/extension'
 
 class <%= class_name %> < Corvid::Plugin
-  # Plugins can be extensions but are not by default.
-  #include Corvid::Extension
 
-  # The path for Ruby to require in order to load this plugin.
-  #
-  # @return [String] The path to require, usually relative to your lib dir.
   require_path '<%= require_path %>'
 
-  # The path to the directory containing the plugin's resources.
-  #
-  # @return [String] An absolute path.
-  resources_path File.expand_path('../../../resources', __FILE__)
-
-  # A manifest of all features provided by the plugin.
-  #
-  # @return [Hash<String,Array<String>>] A hash with keys being feature names, and the values being a 2-element
-  #   array of the feature's require-path, and class name, respectively.
   feature_manifest ({
   })
 
--- test/spec/%name%_plugin_spec.rb.tt	2012-08-14 11:15:55.653476368 +1000
+++ test/spec/%name%_plugin_spec.rb.tt	2012-08-14 11:23:56.566790574 +1000
@@ -5,7 +5,7 @@
 
 describe <%= class_name %> do
   include Corvid::ResourcePatchTests
-  use_resources_path <%= class_name %>.new.resources_path
+  res_patch_dir "#{APP_ROOT}/resources"
 
   include_feature_update_install_tests '<%= name %>', <%= class_name %>.new
 end
--- test/spec/resources_spec.rb	2012-08-14 11:16:26.940408170 +1000
+++ test/spec/resources_spec.rb	2012-08-14 11:23:56.566790574 +1000
@@ -4,7 +4,7 @@
 
 describe "Actual Resources" do
   include Corvid::ResourcePatchTests
-  use_resources_path "#{APP_ROOT}/resources"
+  res_patch_dir "#{APP_ROOT}/resources"
 
   include_patch_validity_tests
 end
